R/helper-functions.R

```diff
@@ -824,7 +824,7 @@ get_colstyle <- function(wb, sheet = current_sheet()) {
   st_ids <- NULL
   if (length(wb$worksheets[[sheet]]$cols_attr)) {
     cols <- wb$worksheets[[sheet]]$unfold_cols()
-    st_ids <- cols$s[cols$s != ""]
+    st_ids <- cols$style[cols$style != ""]
```

Comment:
Not sure about this. This is in an unfolded column.  In rows they use s for styles, but it could be corrected. I’ll look it up. The rest is fine, thanks!

Reply:
Alright, it’s [style](https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.spreadsheet.columns?view=openxml-3.0.1)

Reply:
You were correct in the first place

Reply:
Sorry I went too fast. Will merge when CI green